solidity/contracts/test/BridgeStub.sol

```diff
@@ -7,6 +7,11 @@ import "../bridge/Bridge.sol";
 import "../bridge/Wallets.sol";
 
 contract BridgeStub is Bridge {
+    struct Outpoint {
+        bytes32 fundingTxHash;
+        uint32 fundingOutputIndex;
+    }
```

Comment:
Maybe it makes sense to use `BitcoinTx.UTXO` instead? If not, let's at least rename the fields by cutting the `funding` prefix.

Reply:
Done in ef8e07973a0fcae0610e1ea4d8ef685a623a3975.